modules/visual_script/visual_script_builtin_funcs.h

```diff
@@ -71,8 +71,9 @@ class VisualScriptBuiltinFunc : public VisualScriptNode {
 		MATH_RANDOMIZE,
 		MATH_RANDI,
 		MATH_RANDF,
-		MATH_RANDF_RANGE,
+		MATH_RANDFN,
```

Comment:
To match the other changes you've done it should go after `MATH_RANDF_RANGE` now.

Reply:
Fixed already.